src/base/extensions.ts

```diff
@@ -9,27 +9,31 @@
 import {IPluginDesc} from 'phovea_core';
 import {INamedSet} from 'tdp_core';
 import {CLUEGraphManager} from 'phovea_clue';
+import {IStartMenuSectionTab} from '../internal/menu/tabs/DatasetsTab';
 
 export const EXTENSION_POINT_START_MENU = 'ordinoStartMenuSection';
 
 
 export interface IStartMenuSectionDesc extends IPluginDesc {
   readonly name: string;
+  /**
+   * TODO See if cssClass is still necessary in session cards.
+   */
   readonly cssClass: string;
```

Comment:
I think the CSS class can be removed. Please remove this also in the phovea.ts EPs and SCSS files and everywhere else.